File: lib/data/warung.dart
class Warung {
  final String nama;
  final String lokasi;
  final String deskripsi;
  final String gambar;

  Warung({
    required this.nama,
    required this.lokasi,
    required this.deskripsi,
    required this.gambar,
  });

  factory Warung.fromJson(Map<String, dynamic> json) {
    return Warung(
        nama: json['nama'],
        lokasi: json['lokasi'],
        deskripsi: json['deskripsi'],
        gambar: json['gambar']);
  }
}

File: backend/backend/Go_Rest_API/models/warung.go
package models

type Warung struct {
	Id         int64  `gorm:"primaryKey" json:"id"`
	NamaWarung string `gorm:"type:varchar(255)" json:"nama_warung"`
	Gambar     string `gorm:"type:varchar(255)" json:"gambar"`
	Rating     int64  `gorm:"type:int" json:"rating"`
	Lokasi     string `gorm:"type:varchar(255)" json:"lokasi"`
	Deskripsi  string `gorm:"type:text" json:"deskripsi"`
}
